src/rs.cpp

```diff
@@ -1149,9 +1149,13 @@ HANDLE_EXCEPTIONS_AND_RETURN(, min_severity, file_path)
 void rs2_log_to_callback_cpp( rs2_log_severity min_severity, rs2_log_callback * callback, rs2_error** error ) BEGIN_API_CALL
 {
     // Wrap the C++ callback interface with a shared_ptr that we set to release() it (rather than delete it)
+#if BUILD_EASYLOGGINGPP
```

Comment:
If we have the exception already happening inside log.cpp, then no need to do it here?